docs/docs/Get-Started/get-started-installation.md

```diff
@@ -6,166 +6,228 @@ slug: /get-started-installation
+- [uv](https://docs.astral.sh/uv/getting-started/installation/)
+- A dual-core CPU and at least 2 GB of RAM. More intensive use requires a multi-core CPU and at least 4 GB of RAM.
+- A virtual environment created with [uv](https://docs.astral.sh/uv/pip/environments)
 
-</TabItem>
-</Tabs>
+<details>
+<summary>Need help with virtual environments?</summary>
 
-To install a specific version of the Langflow package, add the required version to the command.
-<Tabs groupId="package-manager">
-<TabItem value="uv" label="uv" default>
+Virtual environments ensure Langflow is installed in an isolated, fresh environment.
+To create a new virtual environment, do the following.
 
-```bash
-uv pip install langflow==1.3.2
+<Tabs groupId="os">
+  <TabItem value="macOS" label="macOS">
+    1. Navigate to where you want your virtual environment to be created, and create it with `uv`.
+Replace `VENV_NAME` with your preferred name for your virtual environment.
+```
+uv venv VENV_NAME
+```
+2. Start the virtual environment.
+```
+source VENV_NAME/bin/activate
 ```
+Your shell's prompt changes to display that you're currently working in a virtual environment.
+```
+(VENV_NAME) ➜  langflow git:(main) ✗
+```
+3. To deactivate the virtual environment and return to your regular shell, type `deactivate`.
+To delete the virtual environment, type `rm -rf VENV_NAME`.
+  </TabItem>
+  <TabItem value="Windows" label="Windows">
+1. Navigate to where you want your virtual environment to be created, and create it with `uv`.
+Replace `VENV_NAME` with your preferred name for your virtual environment.
+```
+uv venv VENV_NAME
+```
+2. Start the virtual environment.
+```shell
+VENV_NAME\Scripts\activate
+```
+Your shell's prompt changes to display that you're currently working in a virtual environment.
+```
+(VENV_NAME) PS C:/users/username/langflow-dir>
+```
+3. To deactivate the virtual environment and return to your regular shell, type `deactivate`.
+To delete the virtual environment, type `Remove-Item VENV_NAME`.
+  </TabItem>
+  </Tabs>
+  <TabItem value="Linux" label="Linux">
+    1. Navigate to where you want your virtual environment to be created, and create it with `uv`.
+Replace `VENV_NAME` with your preferred name for your virtual environment.
+```
+uv venv VENV_NAME
+```
+2. Start the virtual environment.
+```
+source VENV_NAME/bin/activate
+```
+Your shell's prompt changes to display that you're currently working in a virtual environment.
+```
+(VENV_NAME) ➜  langflow git:(main) ✗
+```
+3. To deactivate the virtual environment and return to your regular shell, type `deactivate`.
+To delete the virtual environment, type `rm -rf VENV_NAME`.
+  </TabItem>
+</details>
 
-</TabItem>
-<TabItem value="pip" label="pip">
 
-```bash
-pip install langflow==1.3.2
-```
+<Tabs groupId="os">
+  <TabItem value="macOS" label="macOS">
 
-</TabItem>
-</Tabs>
+    1. To install Langflow, run the following command.
+    ```bash
+    uv pip install langflow
```

Comment:
Can you adjust the indentation on the commands so that they left-align below the start of the text instead of to the left of the number?